README.md

```diff
@@ -29,7 +29,7 @@ The Voyager framework has a small ecosystem of components for building GraphQL b
 
 ## Local Development Setup
 
-The [Local Development Guide](./doc/guides/local-development.md) will help contributors to get started developing Voyager Server.
+The [Local Development Guide](./docs/guides/local-development.md) will help contributors to get started developing Voyager Server.
 
```

Comment:
@wtrocki  looks like this link is invalid too? should i remove mention of local dev guide?

Reply:
I have made this change. I should use resources